src/window.js

```diff
@@ -70,6 +70,12 @@ var ToolboxWindow = GObject.registerClass({
         this._lockscreen.add(this._toolbox_frame);
 
         this.get_style_context().add_class('toolbox-surrounding-window');
+
+        const accelGroup = new Gtk.AccelGroup();
+        accelGroup.connect('r'.charCodeAt(),
+            Gdk.ModifierType.SHIFT_MASK | Gdk.ModifierType.CONTROL_MASK, 0,
+            this._onKeyCombo.bind(this));
+        this.add_accel_group(accelGroup);
```

Comment:
It's probably better to make this a GAction in the window action group, in case we ever need to unlock this remotely (and it's more natural to add a keyboard shortcut for it)